requirements.txt

```diff
@@ -1,4 +1,3 @@
-ansible==2.9.27
+ansible-core==2.15.1
 jmespath==0.10.0
```

Comment:
```jmespath``` is a requirement for Prometheus.
If we use Python 3.9, I don't know if this version 0.10.0 would be compatible.
On my system with Python 3.9, the version 1.0.1 is installed.

Reply:
I added the 1.0.1 version